editor.planx.uk/src/@planx/components/MultipleFileUpload/Modal.tsx

```diff
@@ -152,8 +152,10 @@ const SelectMultiple = (props: SelectMultipleProps) => {
         What does this file show?
       </InputLabel>
       <Select
+        native={true}
```

Comment:
This prop was brought to my attention when inspecting hte DOM for testing (there wasn't actually a `<select>`!). While I ended up using `data-testid` for test selectors, I think we'll still want this prop set to `true` for accessibility/screen reading devices. See this thread: https://stackoverflow.com/a/55185837